cmd/indexer/internal/actions/transaction_test.go

```diff
@@ -0,0 +1,150 @@
+package actions
+
+import (
+	"context"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/golang/mock/gomock"
+	"github.com/mailchain/mailchain/cmd/internal/datastore"
+	"github.com/mailchain/mailchain/cmd/internal/datastore/datastoretest"
+	"github.com/mailchain/mailchain/internal/address/addresstest"
+	"github.com/mailchain/mailchain/internal/protocols"
+	"github.com/mailchain/mailchain/internal/protocols/ethereum"
+	"github.com/pkg/errors"
+)
+
+var txHash = []byte("0x98beb27135aa0a25650557005ad962919d6a278c4b3dde7f4f6a3a1e65aa746c")
+var blockHash = []byte("0x373d339e45a701447367d7b9c7cef84aab79c2b2714271b908cda0ab3ad0849b")
+var txn = &datastore.Transaction{
+	From:      addresstest.EthereumCharlotte,
+	BlockHash: blockHash,
+	Hash:      txHash,
+	Data:      hexutil.MustDecode("0x6d61696c636861696e383162336636383539326431393338396439656432346664636338316331666630323835383962653535303436303532366631633961613436623864333739346337653032616565363563386631373733376361366637333564393565303965366131396636303838366638313239326535373835373133343562386531653466393238326531306433396637316238636639653731613231656336393939333637346634616261643231623831393531646565346665643565666465663334643131303264346333336538626662613330623461343730646162643434653938653262363439346136653862363963393336353864393631393639356633313561356266356262313865363265336266623237363463363335323631616366363730303862353761316262333838353164396132656635353730323861336166373839646537396234346662346130336137653637393037343030376531623237"),
```

Comment:
Would be prefered to use `encodingtest.MustDecodeHexZeroX`  instead of `hexutil.MustDecode` as it reduces the external dependencies for this file.